docs/content/usage/developer-workflow.md

```diff
@@ -94,32 +94,41 @@ cluster from these specs will be annotated with that unique ID.
 ### Setup a Python environment
 
 ```bash
-$ fission env create --spec --name python --image fission/python-env:0.11.0 --builder fission/python-builder:0.11.0
+$ fission env create --spec --name python --image fission/python-env --builder fission/python-builder
 ```
 
 This command creates a YAML file under specs called `specs/env-python.yaml`.
 
 ### Code two functions
 
-One function simply returns a simple web form.  You can download the
-code or copy paste from the contents below:
+We will create two functions in python along with an empty `requirements.txt` file so that builder builds the code.
```

Comment:
Add some explanation texts let users know they should put files like the directory structure below.